pkg/app/bootstrap.go

```diff
@@ -55,32 +56,33 @@ var CommandBootstrap = &cli.Command{
 			Usage:   "Dockerhub mirror to use",
 			Aliases: []string{"m"},
 		},
-		&cli.StringFlag{
-			Name:    "registry-ca-keypair",
-			Usage:   "Specify the ca/key/cert file path for the private registry (format: 'ca=/etc/config/ca.pem,key=/etc/config/key.pem,cert=/etc/config/cert.pem')",
-			Aliases: []string{"ca"},
-		},
 		&cli.StringSliceFlag{
 			Name: "ssh-keypair",
 			Usage: fmt.Sprintf("Manually specify ssh key pair as `publicKey,privateKey`. Envd will generate a keypair at %s and %s if not specified",
 				sshconfig.GetPublicKeyOrPanic(), sshconfig.GetPrivateKeyOrPanic()),
 			Aliases: []string{"k"},
 		},
-		&cli.BoolFlag{
-			Name:  "use-http",
-			Usage: "Use HTTP instead of HTTPS for the registry",
-			Value: false,
-		},
 		&cli.StringFlag{
-			Name:    "registry",
-			Usage:   "Specify the registry to pull the image from",
-			Aliases: []string{"r"},
+			Name:  "registry-config",
+			Usage: "Path to a JSON file containing registry configuration",
```

Comment:
If we're going to pass a JSON file, why not use the buildkit TOML file directly?

I think the main purpose of the CLI is to make it user-friendly. If the situation is complex, it's almost the same as setting up a buildkitd instance manually.

This decision will heavily affect the implementation. What do you think?

cc @gaocegege @RichhLi @ByronHsu

Reply:
How about providing the buildkit TOML directly to envd?

Reply:
Very good point. The problem with directly providing a TOML file is that the Docker mount won't be set up. I think this pr may not even be necessary because we might as well just set up buildkitd manually like you said. We will discuss this.

Reply:
@kemingy @gaocegege Would it be possible to keep the current envd implementation, and the only change we'll make is adding a command that accepts JSON input? And we'll add a check where users can only use one of the commands, not both at the same time. So it will be completely backward compatible.

Reply:
> @kemingy @gaocegege Would it be possible to keep the current envd implementation, and the only change we'll make is adding a command that accepts JSON input? And we'll add a check where users can only use one of the commands, not both at the same time. So it will be completely backward compatible.

I'm okay with it.

Reply:
LGTM!

Will you implement in this PR?

Reply:
Yes, or do you want it in different pr?

Reply:
No, I am just asking. Keeping in this PR LGTM.

Reply:
Hi @RichhLi Do you still need this PR?

Reply:
No, I merged this one: https://github.com/tensorchord/envd/pull/1680